<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>report</class>
 <widget class="QWidget" name="report">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>252</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>506</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>506</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>11</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Report</string>
  </property>
  <property name="windowIcon">
   <iconset resource="maya.qrc">
    <normaloff>:/btn/img/btn/mailBtn24.png</normaloff>:/btn/img/btn/mailBtn24.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(31, 31, 31);</string>
  </property>
  <widget class="QLineEdit" name="edtScript">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>47</y>
     <width>486</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Script or Title</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReport">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>145</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="maya.qrc">
     <normaloff>:/btn/img/btn/mailBtn24.png</normaloff>:/btn/img/btn/mailBtn24.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>190</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="maya.qrc">
     <normaloff>:/btn/img/btn/denialBtn24.png</normaloff>:/btn/img/btn/denialBtn24.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>23</width>
     <height>23</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="edtComment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>486</width>
     <height>86</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(175, 175, 175);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="plainText">
    <string>Comment</string>
   </property>
   <property name="backgroundVisible">
    <bool>false</bool>
   </property>
   <property name="centerOnScroll">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="edtErrorMsg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>145</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(175, 175, 175);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="plainText">
    <string>Error Message</string>
   </property>
   <property name="backgroundVisible">
    <bool>false</bool>
   </property>
   <property name="centerOnScroll">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHelp">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>225</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="maya.qrc">
     <normaloff>:/btn/img/btn/helpBtn24.png</normaloff>:/btn/img/btn/helpBtn24.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtMsg">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>225</y>
     <width>386</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Report: Save</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblUser">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>225</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: 'transparent';</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="maya.qrc">:/user/img/user/_default.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblReport">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: 'transparent';</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="maya.qrc">:/shelf/img/shelf/shelf_bug35.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxReport">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>14</y>
     <width>136</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtBottom</enum>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxScript">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>15</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtBottom</enum>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblScript">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: 'transparent';</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="maya.qrc">:/shelf/img/shelf/shelf_default35.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnShowReport">
   <property name="geometry">
    <rect>
     <x>455</x>
     <y>225</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="maya.qrc">
     <normaloff>:/shelf/img/shelf/shelf_outliner35.png</normaloff>:/shelf/img/shelf/shelf_outliner35.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBefore">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>145</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="maya.qrc">
     <normaloff>:/btn/img/btn/btnArrowLeft48.png</normaloff>:/btn/img/btn/btnArrowLeft48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNext">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>145</y>
     <width>46</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="maya.qrc">
     <normaloff>:/btn/img/btn/btnArrowRight48.png</normaloff>:/btn/img/btn/btnArrowRight48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveToHistory">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>190</y>
     <width>46</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="maya.qrc">
     <normaloff>:/btn/img/btn/btnSave.png</normaloff>:/btn/img/btn/btnSave.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFolder">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>190</y>
     <width>41</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="maya.qrc">
     <normaloff>:/btn/img/btn/folderBtn24.png</normaloff>:/btn/img/btn/folderBtn24.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lblErrorImg">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>145</y>
     <width>126</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(161, 161, 161);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="maya.qrc">:/placeholder/img/placeholder/000.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblErrorCount">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>220</y>
     <width>12</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: 'transparent'; 
color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnErrorImg">
   <property name="geometry">
    <rect>
     <x>373</x>
     <y>148</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="maya.qrc">
     <normaloff>:/btn/img/btn/btnCamera48.png</normaloff>:/btn/img/btn/btnCamera48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
  <zorder>lblErrorImg</zorder>
  <zorder>edtComment</zorder>
  <zorder>edtScript</zorder>
  <zorder>btnReport</zorder>
  <zorder>btnCancel</zorder>
  <zorder>edtErrorMsg</zorder>
  <zorder>btnHelp</zorder>
  <zorder>edtMsg</zorder>
  <zorder>lblUser</zorder>
  <zorder>lblReport</zorder>
  <zorder>cbxReport</zorder>
  <zorder>cbxScript</zorder>
  <zorder>lblScript</zorder>
  <zorder>btnShowReport</zorder>
  <zorder>btnBefore</zorder>
  <zorder>btnNext</zorder>
  <zorder>btnSaveToHistory</zorder>
  <zorder>btnFolder</zorder>
  <zorder>lblErrorCount</zorder>
  <zorder>btnErrorImg</zorder>
 </widget>
 <resources>
  <include location="maya.qrc"/>
 </resources>
 <connections/>
</ui>
